spec.html

```diff
@@ -28474,51 +28474,23 @@ <h1>The Date Constructor</h1>
         <li>is the initial value of the *"Date"* property of the global object.</li>
         <li>creates and initializes a new Date object when called as a constructor.</li>
         <li>returns a String representing the current time (UTC) when called as a function rather than as a constructor.</li>
-        <li>is a single function whose behaviour is overloaded based upon the number and types of its arguments.</li>
+        <li>is a single function whose behaviour differs based upon the number and types of its arguments.</li>
```

Comment:
"single" seems redundant here now, just "is a function" should suffice.

Reply:
Agreed.